### DISCUSSION TITLE
Passing model field value to a nested relation for filtering

### DISCUSSION THREAD

**Discussion by YassinEldeeb** [prisma/prisma#8512] [Q&A] [answered]
So I've a User model that have followers and following users, and Post Model and each post belong to a user.

I want to make a timeline feature which a user can go to and get all of the posts from users he's following but I want to only grab the posts that were created after the user has followed them, what I mean If bob has a post that was created at 1pm and another one that was created at 6pm, and john has followed bob at 4pm he should see only the second post created by bob at 6pm cause that is the only post created after he followed bob and shouldn't get older posts before he has followed him.

The problem is that I want to access the `createdAt` value from the `post` inside of filtering posts by users that I'm following in the `PostWhereInput` like that 

```ts
const feed: Prisma.PostWhereInput = {
  author: {
    followers: {
      some: {
        follower_userId: loggedInUserId,
        createdAt: {
          lte: '${Access Post createdAt field}',
        },
      },
    },
  },
  createdAt: '${be accessed inside of followers filtering}'
}
```
or the other way around by flipping `lte` in followers to `gte` in posts createdAt

Is that possible or there's another solution?

**Comment by ryands17** [comment, marked as answer]
@YassinEldeeb 👋 

Unfortunately referencing fields isn't possible in `where` currently so you would need to use a [raw query](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access) for this. We do have a request for this so it would be great if you could add a 👍 to this so that we know the priority 🙂 
https://github.com/prisma/prisma/issues/5048

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
